adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -126,7 +126,7 @@ workflow AdapterOptimus {
   Int max_cromwell_retries = 0
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.46.0"
+  String pipeline_tools_version = "se-fix-timestamps"
```

Comment:
Is this a temporary change for testing that will be revered before merging up?

Reply:
Yes! When this code is run in the integration test, it will use the pipeline-tools docker image tagged with `se-fix-timestamps` so that the WDL tasks use the code from this PR.

Reply:
Unfortunately there's room for error here because we have to remember to change it to the next version (`v0.47.0`) before merging. This is because the adapter WDLs are in the pipeline-tools repo, but also reference pipeline-tools docker images...so it is pretty complicated at the moment.

Reply:
Since they're workflow inputs it seems like there'd be a way to set these in an options file that's used for testing. But I guess you still need a way to communicate the docker tag required to the test runner.

Reply:
That's true! We talked about doing that a while ago and haven't got around to it yet. But essentially Lira would need to include the version of `pipeline-tools` that it's using in the workflow inputs file that it generates dynamically before starting workflows.